2024/incoming/have-fun-with-map-tube.pod

```diff
@@ -0,0 +1,169 @@
+Author: Mohammad Sajid Anwar <mohammad.anwar@yahoo.com>
+Title: Have fun with Map::Tube
+Topic: Map::Tube
+
+=encoding utf8
+
+=head2 Have fun with Map::Tube
+
+L<Map::Tube|https://metacpan.org/dist/Map-Tube> was one of my earliest submissions to C<CPAN> and it's a project that's very close to my hear. Over the years, I've had the joy of writing about it and I've spoken about it at many conferences. I consider it my baby though I share it with many amazing contributors who have helped make it sleeker and more powerful. Each day, I'm thankful for the contributions of those who've made this project better. Many thanks also to everyone who has used used the library to build maps for cities around the world. As of today, there are L<41 maps|https://metacpan.org/pod/Map::Tube#MAP-LEADER-BOARD> created by C<11 contributors>.
+
+=head2 Inspiration behind the map
+
+Our first window to open is about inspiration! The journey began with the idea to create a map for the C<London Tube Map>, the famous underground rail network. The first map created was L<Map::Tube::London|https://metacpan.org/dist/Map-Tube-London>. As work on C<Map::Tube::London> progressed, I realized there was an opportunity to build a framework so others could create map for their own cities. And so, C<Map::Tube> was born. So now you know where the term C<Tube> came from!
+
+=head2 The first day's play: Let's get started
+
+It's time to open the first door and dive in! To get started, you'll need to install the framework and at least one map. For today, let's stick with the C<London Tube Map>.
+
+=begin code
+
+    $ sudo cpanm -v Map::Tube
```

Comment:
nit: Do we really want to encourage people to use `sudo` to install CPAN stuff? It's a personal bone to pick, but I tell my students that devs should never have to go sudo to install stuff they need, if only to encourage them to *never* use system (perl|python|node|whatever)